packages/klighd-core/src/options/components/option-inputs.tsx

```diff
@@ -56,26 +56,32 @@ interface ChoiceOptionProps extends BaseProps<string> {
     availableValuesLabels?: string[];
 }
 
+/** This Function calls the onChange method, so that a click cause an active change */
+function getCurrentValue(e: Event, props: ChoiceOptionProps) {
```

Comment:
Both the comment nor the function name really reflect what this function does. Always try to think of what the method calculates or executes when looking at it alone, without necessarily knowing about the context it is called from in your implementation. The apparent use of this method is to call the `onChange` function of the given `props` for the given event, if applicable. From this method alone this has nothing to do with "clicks" directly. and `getCurrentValue` sounds like a getter of some sorts, a better name would be `handleChoiceEvent`, `onChoiceEvent` or similar.
Furthermore, we like to include the return type on every method, so this should state `: void` before the opening bracket `{`.